packages/slice-machine/components/Grid/index.tsx

```diff
@@ -6,21 +6,23 @@ interface GridProps<T> {
   renderElem: (elem: T, index: number) => JSX.Element | null;
   defineElementKey: (elem: T) => string;
   gridTemplateMinPx?: string;
+  gridGap?: string;
 }
 
 function Grid<T>({
   elems,
-  gridTemplateMinPx = "320px",
   renderElem,
   defineElementKey,
+  gridTemplateMinPx = "320px",
+  gridGap = "16px",
 }: GridProps<T>): JSX.Element {
   return (
     <Box
       as="section"
       sx={{
         display: "grid",
         gridTemplateColumns: `repeat(auto-fill, minmax(${gridTemplateMinPx}, 1fr))`,
-        gridGap: "16px",
+        gridGap: gridGap,
```

Comment:
the Grid component is used in other places (in the SliceZone and in the SliceZoneUpdate), where the grid row gap should remain 16px, therefore I passed it as an optional prop that is 16px by default.